shuffle/move/examples/e2e/message.test.ts

```diff
@@ -5,10 +5,9 @@
 
 import {
   assert,
-  fail,
+  assertEquals,
 } from "https://deno.land/std@0.85.0/testing/asserts.ts";
-import * as DiemHelpers from "../main/helpers.ts";
-import * as context from "../main/context.ts";
+import { delay } from "https://deno.land/std@0.114.0/async/delay.ts";
```

Comment:
```suggestion
import { delay } from 'https://deno.land/x/delay@v0.2.0/mod.ts';
```
Not sure why but 0.114.0 fails on my local, but 0.2.0 works, saying delay could not be downloaded for 0.114.0

Reply:
I'm inclined to keep the original because it's part of the `std` while the other one is third party. Will dig in more.

Reply:
feel free to continue on with this one if it works for you, might be a localized problem